db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_02_17_162941) do
+ActiveRecord::Schema.define(version: 2023_03_28_153746) do
```

Comment:
can you remove the schema.rb file from your PR since there are no migration changes?
In general this should only be included if you created a migration.